lm_eval/api/metrics.py

```diff
@@ -243,6 +243,15 @@ def perplexity_fn(items):  # This is a passthrough function
     return items
 
 
+@register_metric(
+    metric="perplexity_rolling",
```

Comment:
I'd possibly prefer `token_perplexity` as a name for this metric to make it clear how it differs from the other ones and to better indicate that it's not more of a "default" (imo) than the other normalization approaches.

Reply:
Done.